src/integrationTest/java/uk/gov/hmcts/darts/task/runner/impl/UnstructuredDataExpiryDeletionAutomatedTaskITest.java

```diff
@@ -0,0 +1,598 @@
+package uk.gov.hmcts.darts.task.runner.impl;
```

Comment:
maybe by design but should we have ITest in the name. Should it be Test

Reply:
This is to differentiate the integration tests from the unit tests

Reply:
isnt the fact its under integrationTest good enough? The only reason I mention this is that nothing else does this